### DISCUSSION TITLE
how to authenticate a user with srpring boot

### DISCUSSION THREAD

**Discussion by Yogendra9494** [community/community#88696] [Programming Help] [answered]
### Body

NA

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mohit6350** [comment, marked as answer]
> ### Body
> NA
> 
> ### Guidelines
> * [x]  I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

Use a json web token - JWT

**Reply by Yogendra9494** [reply]
thanks
